docs/sql-migration-guide-upgrade.md

```diff
@@ -24,6 +24,22 @@ license: |
 
 ## Upgrading From Spark SQL 2.4 to 3.0
 
+  - Since Spark 3.0, we upgraded the built-in Hive to 2.3 when enabling the Hadoop 3.2+ profile. This upgrade fixes the following issues:
```

Comment:
Does this need to go in the migration guide? it's helpful info but does it help a user know what they need to do differently?

Reply:
Add to release note?

Reply:
The JIRA lists can be moved to the PR description, but we should document the actual behavior difference in migration guide.

Reply:
OK. Create [SPARK-27686](https://issues.apache.org/jira/browse/SPARK-27686) to update migration guide